tests/phpunit/IntegrationTest.php

```diff
@@ -210,6 +210,28 @@ public function testQueryTimeoutLimit(): void
         }
     }
 
+    public function testSchema(): void
+    {
+        $connection = new Connection([
+            'host' => getenv('SNOWFLAKE_HOST'),
+            'user' => getenv('SNOWFLAKE_USER'),
+            'password' => getenv('SNOWFLAKE_PASSWORD'),
+            'database' => getenv('SNOWFLAKE_DATABASE'),
+            'schema' => $this->sourceSchemaName,
+            'warehouse' => getenv('SNOWFLAKE_WAREHOUSE'),
+        ]);
+
+        $this->assertSame(
```

Comment:
tady bych poprosil komentář co co dělá - po chvíli jsem pochopil, že je to je vlastně test nejdřív, že když to tam dám, tak to tam je a když to tam nedám, tak je tam to destSchemaName, ale není to na první pohled patrné.

Reply:
Jo bude to asi lepsie ked to bude okomentovane :) Pridal som tam komentare je to uz zrozumitelnejsie ?

Reply:
Jo, supr.